scss/product/components/_headers.scss

```diff
@@ -47,12 +52,29 @@
   }
 }
 
+.outlet-header-meta-bar {
+  display: grid;
+  flex-direction: column;
+  justify-content: center;
+  border-left: 1px solid $metadata-line-color;
+  text-align: justify;
+  text-justify: inter-word;
```

Comment:
text-justify is showing `unknown property in chrome`

Reply:
chrome does not support `inter-word` property

Reply:
Hence, removed the property to maintain consistency.